plugins/@grouparoo/hubspot/src/lib/export-contacts/exportRecord.ts

```diff
@@ -124,6 +136,22 @@ export const exportRecord: ExportRecordPluginMethod = async ({
 
       await client.createOrUpdateContact(sortedDataFields);
 
+      // handle associations.
+      if (companyKey) {
+        const oldCompanyKeyValue = oldRecordProperties[companyMappingKey];
+        const newCompanyKeyValue = newRecordProperties[companyMappingKey];
+        if (!contact) {
```

Comment:
can this be returned from `createOrUpdateContact`

Reply:
createOrUpdateContact returns undefined